components/MainNavBar.vue

```diff
@@ -76,7 +76,7 @@
            { name : 'Social Bubble Chart', url : '/', external: false },
          ] },
          { name : 'About', url : '#', external: false, subpages : [
-           { name : 'Overview', url : '/', external: false },
+           { name : 'Overview', url : '/about', external: false },
```

Comment:
wondering about this. On the main site you click on "About" to go to the about section. I think it is better to have an overview instead and not have the dropdown clickable but if we are going down this route, we need to make sure that the "About" in the menu is not a link and that it has some indication that it opens a drop down. Perhaps like a chevron to the right

Reply:
I understand your point as could be confusing to users but all the nav bar headings are dropdowns and none of them use a chevron. Should this be something that is added to all of them?

Reply:
yeah I think this should be added for all dropdowns ideally. Thanks Ruby

Reply:
have added for all dropdowns with subpages

Reply:
Thanks Ruby. Looks good